omod/src/main/resources/config.xml

```diff
@@ -240,7 +232,21 @@
 		</description>
 	</globalProperty>
 
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pixIdentifierTypeUuidList</property>
+		<defaultValue></defaultValue>
+		<description>
+			Specifies sending identifier UUID list (separate with comma)
+		</description>
+	</globalProperty>
 
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pdqIdentifierTypeUuidList</property>
+		<defaultValue></defaultValue>
+		<description>
+			Specifies sending identifier UUID list (separate with comma)
```

Comment:
Specifies the list of identifiers to send in a PDQ query to the MPI. Takes a list of UUID of patient identifier types - separate them with commas.